urls.py

```diff
@@ -1,19 +1,17 @@
-from collections import namedtuple
+from inspect import getmembers, isclass
 
 from django.urls import path, re_path
-from djgeojson.views import GeoJSONLayerView
 from django.views.decorators.cache import cache_page
-from stemp_abw.app_settings import MAP_DATA_CACHE_TIMEOUT
+from djgeojson.views import GeoJSONLayerView
 
-from . import views
-from inspect import getmembers, isclass
 from meta.models import Source
 from meta.views import AppListView, AssumptionsView
-#from stemp_abw.views.serial_views import SplitDataView
+from stemp_abw.app_settings import MAP_DATA_CACHE_TIMEOUT
+from . import views
 
 app_name = 'stemp_abw'
```

Comment:
Shouldn't we set this in app_settings.py?
`APP_NAME = 'stemp_abw'`
and import where needed..

Reply:
As spoken app_name is created by Django. Maybe we find a better solution in the future. But wil leave this solution for right now.